src/components/ec-donut/ec-donut.vue

```diff
@@ -73,47 +73,37 @@
   </div>
 </template>
 
-<script>
+<script setup>
+import { computed } from 'vue';
+
 import EcIcon from '../ec-icon';
 
-export default {
-  name: 'EcDonut',
-  components: { EcIcon },
-  props: {
-    amount: {
-      type: Number,
-      required: true,
-    },
-    used: {
-      type: Number,
-      required: true,
-    },
-  },
-  data() {
-    return {
-      radius: 48,
-    };
+const props = defineProps({
+  amount: {
+    type: Number,
+    required: true,
   },
-  computed: {
-    percentageUsed() {
-      if (this.used > this.amount) {
-        return 100;
-      }
-      if (this.used <= 0) {
-        return 0;
-      }
-      return (this.used / this.amount) * 100;
-    },
-    // Calculate the circumference
-    dashArray() {
-      return 2 * Math.PI * this.radius;
-    },
-    // Calculate the how much needs to be offset of the used circle
-    dashOffset() {
-      return this.dashArray * (1 - this.percentageUsed / 100);
-    },
+  used: {
+    type: Number,
+    required: true,
   },
-};
+});
+
+const radius = 48;
+
+const percentageUsed = computed(() => {
+  if (props.used > props.amount) {
+    return 100;
+  }
+  if (props.used <= 0) {
+    return 0;
+  }
+  return (props.used / props.amount) * 100;
+});
+// Calculate the circumference
+const dashArray = computed(() => 2 * Math.PI * radius);
```

Comment:
Returned value is not an array. It is better to name it "circumference" and remove the comment above.

Reply:
I think dasharray here is referring to the svg element (stronke-dasharray and dashoffset), not an actual "array". But let me know if you think it's better to rename it.

Reply:
I see :). Maybe it is better to rename both "array" vars and remove their comments, just to make the script code cleaner.

circumference
usedArcLength